ee/clickhouse/sql/person.py

```diff
@@ -153,15 +158,16 @@
 """
 ).format(
     table_name=PERSON_STATIC_COHORT_TABLE,
+    cluster=CLICKHOUSE_CLUSTER,
     engine=table_engine(PERSON_STATIC_COHORT_TABLE, "_timestamp", REPLACING_MERGE_TREE),
     storage_policy=STORAGE_POLICY,
     extra_fields=KAFKA_COLUMNS,
 )
 
 DROP_PERSON_STATIC_COHORT_TABLE_SQL = """
-DROP TABLE {}
+DROP TABLE {} ON CLUSTER {}
```

Comment:
Nitpicking: Instead of blank `{}` use f-strings.

Reply:
I 100% agree. I'm not a fan of `{}` was just using what was there